src/wp-includes/class-wp-theme-json-schema.php

```diff
@@ -38,10 +38,11 @@ class WP_Theme_JSON_Schema {
 	 * @since 6.6.0 Migrate up to v3.
 	 *
 	 * @param array $theme_json The structure to migrate.
-	 *
+	 * @param string $origin    Optional. What source of data this object represents.
+	 *                          One of 'blocks', 'default', 'theme', or 'custom'. Default 'theme'.
 	 * @return array The structure in the last version.
 	 */
-	public static function migrate( $theme_json ) {
+	public static function migrate( $theme_json, $origin = 'theme' ) {
```

Comment:
Update`@since 6.6.0` that mention new variable

Reply:
👍 https://github.com/WordPress/wordpress-develop/pull/6737/commits/f57351c24a51c467c15a1a81080968b1d6e77dfb